cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -567,9 +539,9 @@ func TestGetAPIServerCommand(t *testing.T) {
 				"--secure-port=123",
 				"--allow-privileged=true",
 				"--kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname",
+				"--experimental-bootstrap-token-auth=true",
```

Comment:
These multi-part cleanup PRs have gotta stop.  Each change should stand on their own, and in case we need to revert it's not a weird revert that cuts across areas.